src/Components/Cart/cart.css

```diff
@@ -0,0 +1,311 @@
+  background-color: var(--pink-color);
+}
+
+.place-order {
+  background-color: var(--bg-color);
+}
+
+.cart-wishlist-btn {
+  border: 1px solid var(--bg-color);
+  background: var(--white);
+  color: var(--bg-color);
+}
+
+.cart-wishlist-btn:hover {
+  background-color: var(--bg-color);
+  color: var(--white);
+}
+
+.cart-product-img {
+  padding: 1.5rem 1rem 1.5rem 1.5rem;
+  object-fit: cover;
+}
+
+.cart-product-off {
+  color: var(--pink-color);
+  font-weight: bold;
+  padding: 0 10px;
+}
+
+.cart-product-btn {
+  padding: 5px;
+  margin: 5px;
+  border-radius: 5px;
+  border: 1px solid var(--bg-color);
+  cursor: pointer;
+  background-color: var(--white);
+}
+
+.cart-product-btn:hover {
+  background-color: var(--bg-color);
+  color: var(--white);
+}
+
+.flex {
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
+}
+
+.cart-product-number {
+  border: 1px solid black;
+  padding: 5px;
+}
+
+.price-details-card {
+  background-color: var(--white);
+  padding: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  row-gap: 0.7rem;
+  /* width: 100%; */
+}
+
+.flex-row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.footer {
+  background-color: var(--pink-color);
+  color: var(--white);
+  padding: 1rem;
+}
+
+.footer-items {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  row-gap: 1rem;
+  font-size: 1.1rem;
+}
+
+.footer-icon {
+  color: var(--bg-color);
+}
+
+.footer-social-icon {
+  color: var(--white);
+  font-size: 1.5rem;
+  margin: 0 0.5rem;
+}
+
+.nav-link,
+.nav-link:hover {
+  color: var(--bg-color);
+}
+
+.active {
+  color: var(--pink-color);
+}
+
+.address {
+  padding: 0.6rem;
+  background-color: var(--white);
+  margin: 0 0 1rem 0;
+  width: 60%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 1rem;
+}
+
+.address-title {
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.detail-address {
+  font-size: 1rem;
+}
+
+.click-btn {
+  margin-left: auto;
+  padding: 0.1rem 0.7rem;
+  color: var(--pink-color);
+  border-color: var(--pink-color);
+  background: var(--white);
+}
+
+.click-btn:hover {
+  background: var(--pink-color);
+  color: var(--white);
+}
+
+.address-sec {
+  width: 70%;
+}
+
+.modal {
+  display: none;
+  overflow: hidden;
+}
+
+.modal-content {
+  background-color: var(--product-bgcolor);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  width: 40%;
+  height: auto;
+  border-radius: 5px;
+}
+
+.input {
+  width: 20rem;
+  margin: 0.5rem 0;
+}
+
+.input-address {
+  height: 7rem;
+  width: 20rem;
+}
+
+.name-label {
+  font-size: 1.1rem;
+  color: var(--bg-color);
+  font-weight: 500;
+  margin-bottom: 1rem;
+}
+
+.cancel-btn {
+  background-color: var(--product-bgcolor);
+  color: var(--bg-color);
+  border-radius: 5px;
+  border: 1px solid var(--bg-color);
+}
+
+.save-btn {
+  background-color: var(--pink-color);
+  border-radius: 5px;
+}
```

Comment:
If this is all related to address then try to have a separate component for address and a separate CSS file because this is a huge file and any changes you want to make in this will be difficult.